lib/msf/core/rpc/v10/rpc_db.rb

```diff
@@ -276,15 +276,31 @@ def rpc_creds(xopts)
           port = login.service.port.to_i
           proto = login.service.proto.to_s
         end
-        ret[:creds] << {
+
+        if cred.private.nil?
+            ret[:creds] << {
+                :user => cred.public.username.to_s,
+                :pass => nil,
+                :updated_at => nil,
```

Comment:
This will need to be `cred.public.updated_at` since that is the main part of the credential here.